.github/scripts/binutils/build.sh

```diff
@@ -39,6 +39,16 @@ if [[ "$RUN_CONFIG" = 1 ]] || [[ ! -f "$BINUTILS_BUILD_PATH/Makefile" ]]; then
                 ;;
         esac
 
+        case "$ARCH-$PLATFORM" in
+            aarch64-*cygwin*)
+                # ADDED: --enable-targets=aarch64-pep
+                # ADDED: --disable-sim
+                TARGET_OPTIONS="$TARGET_OPTIONS \
+                    --enable-targets=aarch64-pep \
```

Comment:
I wonder why this is not needed to be specified explicitly for `x86_64` build while it is for `aarch64` build.